client/package.json

```diff
@@ -241,7 +241,8 @@
         {
           "id": "bitbakeRecipes",
           "name": "Recipes",
-					"contextualTitle": "Recipes Explorer"
+					"contextualTitle": "Recipes Explorer",
+          "when": "resourceLangId == bitbake"
```

Comment:
So this is checking the language id. When you switch to another file type, say, JSON, the icon disappears. Don't know about others, but seeing the icon appearing and disappearing is a little confusing and annoying to me.

Reply:
Thanks for testing this! In fact I noticed this erroneous behavior but fixed it in the next PR: https://github.com/savoirfairelinux/vscode-bitbake/pull/63/commits/af0dad6be377dd56dc05f7e0cd97f7e76e504c80

Reply:
Sounds good.